test/app-tap/tarantoolctl.test.lua

```diff
@@ -142,7 +162,7 @@ local function check_ok(test, dir, cmd, args, e_res, e_stdout, e_stderr)
         ares = ares and val
     end
     if e_stdout ~= nil then
-        local val = test:is(res, e_res, ("check '%s' stdout for '%s'"):format(cmd,args))
+        local val = test:ok(stdout:find(e_stdout), ("check '%s' stdout for '%s'"):format(cmd,args))
```

Comment:
String too long, limit it in 72 chars:
echo "        local val = test:ok(stdout:find(e_stdout), ("check '%s' stdout for '%s'"):format(cmd,args))" | wc -c
94

Reply:
Fixed.